### PR TITLE
Tracked the throttling policy ID - Reviewed code.

### PR DESCRIPTION
**Pull request by andralungu**
Addressed the code review comments from https://github.com/adobe-apiplatform/api-gateway-request-tracking/pull/12#issuecomment-320639708

@cristianconstantin, @atrifan ping.

### CODE REVIEW

File: src/lua/api-gateway/tracking/validator/blockingRulesValidator.lua [hunk 1]
```diff
@@ -54,7 +54,7 @@ function _M:validate_blocking_rules(config_obj)
         return ngx.HTTP_OK, ""
     end
 
-    ngx.var.request_blocked = blocking_rule.id;
+    ngx.var.blocked_by = math.floor(tonumber(blocking_rule.id/10000));
```
Comment (atrifan) [inline src/lua/api-gateway/tracking/validator/blockingRulesValidator.lua]: it's tonumber(blocking_rule.id)/10000

File: src/lua/api-gateway/tracking/validator/delayingRulesValidator.lua [hunk 2]
```diff
@@ -55,7 +55,7 @@ function _M:validate_delaying_rules(config_obj)
 
     ngx.var.request_delayed = actualDelay
     ngx.log(ngx.DEBUG, "delaying request with " .. tostring(actualDelay) .. " seconds out of the rule setting: " .. tostring(delaying_rule.data) .. " seconds")
-    ngx.var.request_delayed = delaying_rule.id;
+    ngx.var.delayed_by = math.floor(tonumber(delaying_rule.id/10000));
```
Comment (atrifan) [inline src/lua/api-gateway/tracking/validator/delayingRulesValidator.lua]: tonumber(delaying_rule.id)/10000

File: src/lua/api-gateway/tracking/validator/rewritingRulesValidator.lua [hunk 3]
```diff
@@ -58,7 +58,7 @@ function _M:validate_rewrite_rules(config_obj)
         return nil
     end
 
-    ngx.var.request_rewritten = rewrite_rule.id;
+    ngx.var.rewritten_by = math.floor(tonumber(rewrite_rule.id/10000))
```
Comment (atrifan) [inline src/lua/api-gateway/tracking/validator/rewritingRulesValidator.lua]: tonumber(rewrite_rule.id)/10000

**Review by atrifan — CHANGES_REQUESTED** [review]

**Comment by andralungu** [comment]
```10.76.208.236 [08/Aug/2017:08:33:00 +0000] host="throttling-test-dev.adobe.io" request="GET /hello HTTP/1.1" api_key="free_api_key" status=200 bbs=14 rt=5.500 hr="-" ua="-" xfwdf="193.105.140.131" upadd="-" upstat=- uprt="-" sid="it" sname="it" region="ue1" reqid="DaLJTQGNNaiZC5nLk0Nitc10cdfLcl14" delayed="5.5" delayed_by="47" blocked_by="" rewritten_by=""``` tested on dev.

**Pull request merged**